shoper.py

```diff
@@ -0,0 +1,135 @@
+            database.at[row, 'active'] = 0
+            print(database['name'][row],"at code", database['product_code'][row])
+    
+
+def repair_dateprices(database, months):
+    """
+    Repairs date prices (10.99 [meant] -> oct.99 [edited] -> 10.99)
+    caused by editing the .csv file in Excel, while Shoper needs the meant ones
+    
+    csv_file: .csv file to work with
+    months: dictionary of area-specified month names e.g. 'Jan':'1'
+    column_types: column types ensuring proper encoding of file
+    """
+
+    print("Replaced following prices:")
+    
+    for row in range(database.shape[0]):
+        
+        price_value = str(database['price'][row])
+        price = price_value[:3]
+
+        if price in months.keys():
+            
+            new_price = price_value.replace(price, months[price])
+            database.at[row, 'price'] = new_price
+            print(database['price'][row], "at code", database['product_code'][row])
+
+
+def linktoimage(database, folder_address, url_address):
+    """
+    Connects created links to exact .csv records - e.g. item codes
+    If more e.g. photo names are in format "code.xxx", "code_1.xxx"
+    they can be connected to additional fields like image_1, image_2 etc.
+    
+    csv_file: .csv file to work with
+    folder_address: address of a folder containing photo folders
+    column_types: column types ensuring proper encoding of file
+    """
+
+    folders = []
+    sorted_pl = []
+    
+    for folder in os.listdir(folder_address):  # folder in list of folders
+        if os.path.splitext(folder)[1] == "":  # no extension == folder
+            folders.append(folder)
+
+    for folder in folders:
+        path = folder_address + folder
+
+        for file in os.listdir(path):
+            file_name = os.path.splitext(file)[0]
+            image_prefix = f'{str(file_name)[:-2]}_'
+            
+            if search(image_prefix, file_name)!=None:
+                
+                if int(file_name[:-2]) in database['product_code'].values:
+                    
+                    sorted_pl.append([url_address + folder + "/" + file,
+                                      int(file_name[:-2]), int(file_name[-1])])
+    
+            if int(file_name) in database['product_code'].values:
+                
+                sorted_pl.append([url_address + folder + "/" + file,
+                                  int(file_name), 0])
+    
+    sorted_pl = sorted(sorted_pl, key=itemgetter(1, 2))
+    product_no = 1
+        
+    for photolink in range(len(sorted_pl)):
+        
+        image_number = sorted_pl[photolink][2] + 1
+        imageno_string = f'images {image_number}'
+        
+        if photolink!=0:
+            
+            if sorted_pl[photolink][1]!= sorted_pl[photolink-1][1]:
+                product_no = product_no + 1
+       
+        database.at[product_no -1, imageno_string] = sorted_pl[photolink][0]
+
+
+months_PL = {'sty': '1', 'lut': '2', 'mar': '3', 'kwi': '4',
```

Comment:
To jest globalna zmienna, więc powinna być napisana z wielkich liter.
Do tego masz ich dużo, więc najlepiej, jakby to był osobny plik np. `constants.py` 
Jeśli nie chcesz innego pliku, to takie zmienne globalne powinny być na górze pliku.